packages/create-accurapp/index.js

```diff
@@ -123,6 +123,10 @@ const packageJson = {
     start: 'accurapp-scripts start',
     build: 'accurapp-scripts build',
   },
+  browserslist: {
+    production: ['last 2 versions', 'ie 11'],
```

Comment:
Take a read over this blog post: https://jamie.build/last-2-versions

Reply:
@markov00 already done, check the Files Changed not the commits please. This is outdated